runtime/ops/worker_host.rs

```diff
@@ -483,7 +464,7 @@ fn op_create_worker(
 
   // Setup new thread
   let thread_builder =
-    std::thread::Builder::new().name(format!("deno-worker-{}", worker_id));
```

Comment:
Please revert - otherwise there will only be a number in the thread name

Reply:
@bartlomieju please let me know if this is ok, or if I should still change it

Reply:
I see, that sounds good